src/main/java/org/zalando/nakadi/repository/db/SchemaRepository.java

```diff
@@ -0,0 +1,49 @@
+package org.zalando.nakadi.repository.db;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+import org.zalando.nakadi.domain.EventTypeSchema;
+
+import java.io.IOException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+@Component
+public class SchemaRepository extends AbstractDbRepository {
+
+    @Autowired
+    public SchemaRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper objectMapper) {
+        super(jdbcTemplate, objectMapper);
+    }
+
+    public List<EventTypeSchema> getSchemas(final String name, final int offset, final int limit) {
+        return jdbcTemplate.query(
+                "SELECT ets_schema_object FROM zn_data.event_type_schema " +
+                        "WHERE ets_event_type_name = ? ORDER BY ets_schema_object->>'version' DESC LIMIT ? OFFSET ? ",
```

Comment:
Are you sure that the ordering will be correct? I think it will order wrongly for the following schemas:
"1.0.2"
"2.10.3"
"10.0.0"